.github/workflows/main.yml

```diff
@@ -15,21 +15,34 @@ jobs:
       matrix:
         pair:
           - erlang: master
+            dialyzer: true
+            rebar3: 3.16.0
           - erlang: '24.0'
+            dialyzer: true
+            rebar3: 3.16.0
           - erlang: 23.3
+            rebar3: 3.6.1
           - erlang: 22.3
+            rebar3: 3.6.1
```

Comment:
We could pull `3.16.0` till this one, but it won't support previous versions any more.